build.sh

```diff
@@ -0,0 +1,11 @@
+mvn clean install
+
+# opendsscmd build product from fpc
+rm -rf distrib
+mkdir distrib
+cp ../OpenDSS/Source/CMD/test/opendsscmd distrib
+cp ../linenoise-ng/build/liblinenoise.so distrib
+
+./stop.sh
+docker rmi temcderm/cimhub:0.0.1
+sudo docker build -t="temcderm/cimhub:0.0.1" .
```

Comment:
why is sudo required here to build docker?  is the temcderm going to be the final location for these or should it have gridappsd label instead?